packages/opentelemetry-remote-sampler-aws-xray/package.json

```diff
@@ -0,0 +1,71 @@
+{
+  "name": "@opentelemetry/remote-sampler-aws-xray",
+  "version": "0.34.0",
+  "description": "OpenTelemetry remote sampler for AWS X-Ray",
+  "keywords": [
+    "aws",
+    "nodejs",
+    "opentelemetry"
+  ],
+  "license": "Apache-2.0",
+  "author": "OpenTelemetry Authors",
+  "bugs": {
+    "url": "https://github.com/open-telemetry/opentelemetry-js-contrib/issues"
+  },
+  "main": "build/src/index.js",
+  "types": "build/src/index.d.ts",
+  "files": [
+    "build/src/**/*.js",
+    "build/src/**/*.js.map",
+    "build/src/**/*.d.ts",
+    "doc",
+    "LICENSE",
+    "README.md"
+  ],
+  "repository": "open-telemetry/opentelemetry-js-contrib",
+  "scripts": {
+    "clean": "rimraf build/*",
+    "compile": "tsc -p .",
+    "lint": "eslint . --ext .ts",
```

Comment:
I think this needs some configuration files like `.eslintignore`, `.eslintrc.js`. Did you try to run this command to verify it is working correctly?